ingress/controllers/nginx-third-party/controller.go

```diff
@@ -450,9 +450,7 @@ func (lbc *loadBalancerController) Run() {
 	go lbc.svcController.Run(lbc.stopCh)
 
 	// periodic check for changes in configuration
-	go wait.Until(lbc.sync, 5*time.Second, wait.NeverStop)
-
-	time.Sleep(5 * time.Second)
+	go wait.Until(lbc.sync, 10*time.Second, wait.NeverStop)
```

Comment:
hmm this is confusing, why are you calling lbc.sync here? shouldn't you just startup the controller and let it sync on each watch/relist?